references/kubernetes/cloud-resources/aws/dynamodb.yaml

```diff
@@ -4,34 +4,20 @@ apiVersion: dynamodb.services.k8s.aws/v1alpha1
 kind: Table
 metadata:
   annotations:
+    velocity.tech.v1/exports-name: path="spec.tableName"
     velocity.tech.v1/id: dynamo
-  name: "my-table"
+  name: "{velocity.v1.generate:cloudResourcePrefix}-my-table-var-service-id"
```

Comment:
How does the var-service-id and cloudResourcePrefix work together?

Reply:
Its not the spec its just the metadata that UI based on it. I talked about it with @IamShobe we know we have this constrain.